packages/next/build/analysis/get-page-static-info.ts

```diff
@@ -38,12 +39,16 @@ export async function getPageStaticInfo(params: {
     const { ssg, ssr } = checkExports(swcAST)
     const config = tryToExtractExportedConstValue(swcAST, 'config') || {}
 
-    const runtime =
-      config?.runtime === 'edge'
-        ? 'edge'
-        : ssr || ssg
-        ? config?.runtime || nextConfig.experimental?.runtime
-        : undefined
+    let runtime = ['experimental-edge', 'edge'].includes(config?.runtime)
```

Comment:
Supporting both `'experimental-edge'` and `'edge'`, to keep compatibility with the RSC tests. WDYT @shuding @cramforce

Reply:
Works for me. I mostly care about what we document.